Cake.BenchmarkDotNet/NotANumberNumericConverter.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Newtonsoft.Json;
+using System;
```

Comment:
Please move the system usings to the top of the file.

Reply:
Fixed